.eslintrc.json

```diff
@@ -140,6 +140,10 @@
         "prefer-template": "warn",
         "radix": "warn",
         "yoda": "warn",
+        "n/no-unsupported-features/node-builtins": "error",
+        "n/no-process-exit": "warn",
```

Comment:
Why are not all on warning? The CI anyway fails

Reply:
The idea is to have `error` for possible bugs and `warn` for code style preferences.